src/components/Pages/Service.js

```diff
@@ -71,44 +71,44 @@ const Service = ({ service, intl }) => {
             <div className="wrapper container-fluid">
               <div className="row">
                 <div className="col-xs-12 col-md-10">
-                  {(steps && !!steps.length) && (
+                  {steps &&
+                    !!steps.length &&
                     //just 1 step? don't display steps in list (ul)
-                    steps.length === 1 ? (
+                    (steps.length === 1 ? (
                       <Steps steps={steps} />
                     ) : (
                       <Fragment>
                         <Steps steps={steps} />
                       </Fragment>
-                    )
-                  )}
+                    ))}
 
                   {!!dynamicContent &&
                     dynamicContent.map(content => (
                       <ApplicationBlock key={content.id} content={content} />
                     ))}
 
                   {additionalContent && (
-                    <HtmlFromAdmin
-                      content={additionalContent}
-                    />
+                    <HtmlFromAdmin content={additionalContent} />
                   )}
-                  <div className="coa-Page__contacts-mobile">
-                    {!!contacts &&
-                      !!contacts.length && (
-                        <ContactDetails contacts={contacts} />
-                      )}
-                  </div>
                 </div>
               </div>
             </div>
           </div>
           <div className="coa-Page__side-content">
             <div className="coa-ServicePage__contacts-desktop">
-              {!!contacts &&
-                !!contacts.length && <ContactDetails contacts={contacts} />}
+              {!!contacts && !!contacts.length && (
+                <ContactDetails contacts={contacts} />
+              )}
             </div>
           </div>
         </div>
+        <div className="coa-Page__contacts-mobile">
```

Comment:
moved this out of the man `col`s to make sure we can have properly rendered full width background colors for it